gcloud/_helpers.py

```diff
@@ -30,9 +30,13 @@
 except ImportError:
     app_identity = None
 try:
-    from grpc.beta import implementations
+    from google.gax.grpc import exc_to_code as beta_exc_to_code
+    import grpc
+    from grpc._channel import _Rendezvous
```

Comment:
As the underscores indicate, you're transgressing the supported API.

You should be catching `grpc.RpcError` raised by methods publicly documented to raise `grpc.RpcError`. Of course the instance that you catch may be an instance of a subclass, but that should be of zero consequence to you as an application developer (please loudly let me know if it's not!).